src/svm/svc.rs

```diff
@@ -251,7 +245,7 @@ impl<'a, TX: Number + RealNumber, TY: Number + Ord, X: Array2<TX> + 'a, Y: Array
     pub fn fit(
         x: &'a X,
         y: &'a Y,
-        parameters: &'a SVCParameters<'a, TX, TY, X, Y>,
+        parameters: &'a SVCParameters<TX, TY, X, Y>,
```

Comment:
this change looks right, but we can be sure that all the attributes inside `SVCParameters` have the same lifetime of `&'a`? No, because Kernel has no lifetime (aka its lifetime is `'static` )

Reply:
I thik that `'static` in this case has a different meaning than 'static lifetime